server/vcs.go

```diff
@@ -173,31 +178,35 @@ func (s *Server) composeVCSByID(ctx context.Context, id int) (*api.VCS, error) {
 	vcsFind := &api.VCSFind{
 		ID: &id,
 	}
-	vcs, err := s.VCSService.FindVCS(ctx, vcsFind)
+	vcsRaw, err := s.VCSService.FindVCS(ctx, vcsFind)
 	if err != nil {
 		return nil, err
 	}
 
-	if vcs != nil {
-		if err := s.composeVCSRelationship(ctx, vcs); err != nil {
+	if vcsRaw != nil {
+		vcs, err := s.composeVCSRelationship(ctx, vcsRaw)
+		if err != nil {
 			return nil, err
 		}
+		return vcs, nil
 	}
-	return vcs, nil
+	return nil, nil
 }
 
-func (s *Server) composeVCSRelationship(ctx context.Context, vcs *api.VCS) error {
+func (s *Server) composeVCSRelationship(ctx context.Context, raw *api.VCSRaw) (*api.VCS, error) {
 	var err error
```

Comment:
nit: not related to this change, we don't need to pre-define error if possible.

Reply:
@d-bytebase 
WDYT of this change https://github.com/bytebase/bytebase/pull/792/files#diff-0c812e31ab390c0fee9f18e8423f33c1f9fa59e3a54742a9470275604723fcf8R609?

There are multiple occurrences of pre-defined error in the codebase. If this is prefered, I could make them more consistent.

Reply:
Yeah, either way works for me. Thanks.